Codeinwp/otter-blocks
# Add support for taxonomies in Dynamic Content

Closes #1559 .


### Summary


This feature will show taxonomies that are neither a tag nor a category, usually from 3rd-party plugins.

### Screenshots 

https://github.com/Codeinwp/otter-blocks/assets/17597852/0d2e85cb-df44-4774-a9ba-d525c2a7d77a

----

### Test instructions


1. Create some custom taxonomy. In the video I used this plugin: https://wordpress.org/plugins/custom-post-type-ui/
2. Attack the create taxonomy to the Post.
3. Insert a dynamic content tag for Post Term
4. Select Taxonomy and see the result.

ℹ️ Dynamic content is quite wonky in the preview (it can be seen in the video). So better publish the page to see the results.



---- 

### Checklist before the final review
